gestalt/__init__.py

```diff
@@ -171,6 +148,16 @@ def build_config(self) -> None:
 
         self.__conf_data = self.__flatten(self.__conf_data,
                                           sep=self.__delim_char)
+        self.interpolate()
+
+    def interpolate(self):
+        for k, v in self.__conf_data.items():
+            if isinstance(v, str):
+                if re.match("^ref+.*$", v):
```

Comment:
I think we should elaborate on this regex a bit. This will work, but a little too fast and loose.

Reply:
Updated the regex as per what you suggested